extension/package.json

```diff
@@ -52,7 +53,12 @@
         }
       }
     },
-    "commands": [],
+    "commands": [
+      {
+        "command":"onlythemes.forgetMe",
```

Comment:
Let's add:
```
"category": "Only Themes",
```

This will allow us to group all future commands as "Only Themes: {command title}"

Reply:
It dawns on me. We need to update the extensions README to add how to forget yourself.

Reply:
@MichaelJolley done and done!